_source/_posts/2022-04-07-where-okta-meets-devops.md

```diff
@@ -0,0 +1,68 @@
+
+Additionally, Okta is working on [more new products](https://www.okta.com/press-room/press-releases/okta-introduces-new-okta-privileged-access-product-to-strengthen-security/) for privileged access management, which will be interesting in the DevOps use case. 
+
+## Introspecting Okta
+
+When DevOps refers to a catch-all role focused on the care and feeding of infrastructure, monitoring and alerting fall under its purview. 
+
+So far, we've talked about information flowing from Okta to your software and services, answering questions like "is this login request legitimate?". Automating Okta administration tasks with Terraform pushes configuration from the user to Okta. There's one more direction that information can move relative to your identity provider, however: a stream of metadata about what the identity provider is doing can flow out of the provider and into your monitoring and alerting tools. 
+
+Okta supports a variety of [integrations](https://support.okta.com/help/s/article/Exporting-Okta-Log-Data) for monitoring its various moving parts. For instance, integrations can be set up to exfiltrate Okta logs to [DataDog](https://docs.datadoghq.com/integrations/okta/), [Splunk](https://www.okta.com/integrations/splunk-add-on-for-okta/), or [SumoLogic](https://www.okta.com/integrations/sumologic/). These solutions can combine information about your Okta account with information from your other systems in ways limited only by your imagination: Would you like to get an alert when a user who has recently changed their password accesses a particular resource? How about if the total login failures involving a particular account, across Okta and your other systems, exceed a particular threshold? Would postmortems be easier if every user action logged by Okta showed up alongside the other data in your favorite monitoring tool?
+
+## In Conclusion
+
+Although the field itself is ambiguously defined, DevOps tools and practices constantly interface with identity. DevOps done right ensures that the people can access the resources they should, and not the ones they shouldn't. DevOps practitioners can use Okta to control identity-related behavior of their infrastructure through its many integrations, and can in turn customize and manage their Okta instances by using Terraform and Okta's metrics and monitoring APIs. 
+
+If you work in DevOps, you can't escape the responsibility of managing identity as it relates to your infrastructure. But you can choose how to manage it: Will your organization support you in delegating these tasks to specialists, do you have a use case where it's better to attempt to reinvent these wheels yourself?
```

Comment:
```suggestion
If you work in DevOps, you can't escape the responsibility of managing identity as it relates to your infrastructure. But you can choose how to manage it: Will your organization support you in delegating these tasks to specialists? Do you have a use case where it's better to attempt to reinvent these wheels yourself?
```
This is an important idea, a key summary of your argument. I'd recommend expanding it into 2 separate questions, giving the reader more room to absorb it... Also possible to add "Or" at the opening: "Or do you have a use case..."